libs/clients/directorate-of-immigration/src/lib/directorateOfImmigrationClient.service.ts

```diff
@@ -70,202 +50,141 @@ class DirectorateOfImmigrationClient {
     return this.countryOfResidenceApi.withMiddleware(new AuthMiddleware(auth))
   }
 
-  private criminalRecordApiWithAuth(auth: Auth) {
-    return this.criminalRecordApi.withMiddleware(new AuthMiddleware(auth))
-  }
-
   private residenceAbroadApiWithAuth(auth: Auth) {
     return this.residenceAbroadApi.withMiddleware(new AuthMiddleware(auth))
   }
 
-  private staticDataApiWithAuth(auth: Auth) {
-    return this.staticDataApi.withMiddleware(new AuthMiddleware(auth))
-  }
-
-  private studyApiWithAuth(auth: Auth) {
-    return this.studyApi.withMiddleware(new AuthMiddleware(auth))
-  }
-
   private travelDocumentApiWithAuth(auth: Auth) {
     return this.travelDocumentApi.withMiddleware(new AuthMiddleware(auth))
   }
 
   // Common:
 
-  async getCountries(): Promise<Country[]> {
-    try {
-      const res = await this.optionSetApi.apiOptionSetLookupTypeGet({
-        lookupType: LookupType.Countries,
-      })
+  async getCountries(): Promise<OptionSetItem[]> {
```

Comment:
As you are only forwarding the response you can just do
```suggestion 
   getCountries(): Promise<OptionSetItem[]> {
       return this.optionSetApi.apiOptionSetLookupTypeGet({
           lookupType: LookupType.Countries,
       })
   }
```